tests/monitors/statsd/statsd_test.py

```diff
@@ -41,3 +46,27 @@ def test_statsd_monitor():
             "Didn't get statsd.test metric"
         assert wait_for(p(has_datapoint_with_dim, backend, "foo", "bar")),\
             "Didn't get foo dimension"
+
+
+@pytest.mark.k8s
+@pytest.mark.kubernetes
+def test_statsd_in_k8s(agent_image, minikube, k8s_test_timeout, k8s_namespace):
+    monitors = [
+        {"type": "collectd/statsd",
+         "listenAddress": "127.0.0.1",
+         "listenPort": 8125,
+         "counterSum": True,
+         "deleteSets": True,
+         "deleteCounters": True,
+         "deleteTimers": True,
+         "deleteGauges": True},
+    ]
+    run_k8s_monitors_test(
```

Comment:
How does this work without sending any test udp message to statsd like the above test?

Reply:
I send udp messages via netcat within the minikube container (I forgot to move this from tests.kubernetes.utils.run_k8s_monitors_test when the test was restructured).